test/types/fixed.js

```diff
@@ -0,0 +1,11 @@
+const test = require('ava')
+const fixed = require('../../src/types/fixed')
+
+test('Type: fixed', (t) => {
+  t.true(fixed.isType('fixed'), 'Fixed should default to 256x80')
+  t.true(fixed.isType('fixed8x11'))
+  t.true(fixed.isType('fixed248x35'))
+  t.false(fixed.isType('fixed256x89'), 'Maximum length of N should be 80')
+  t.false(fixed.isType('fixed266x33'), 'Maximum length of M should be 80')
```

Comment:
This `80` should be a `256`. Besides, a test for the minimum could be added.

Reply:
sorry for the typo, adding the requested test